src/docs/guides/BestPracticesForDataAnnotation.md

```diff
@@ -23,29 +23,32 @@ In this guide, we will take a closer look at some experimental scenarios that ev
 
-You should use the Factor building block in such a case to annotate the time after exposure and thereby the sampling point in the isa.study.xlsx file, as this time period will ultimately result in the given output, when all remaining parameters for treatment and analysis were identical.
+<img src="./../img/ISA_AnnotationPattern_Replicates.drawio.svg" style="width:100%;display: block;margin: auto; padding: 30px 0px;">
 
-## Annotation of mixed samples
-This example can be of relevance when you are carrying out labeling experiments or when you are spiking your samples with an internal standard for absolute quantification. The isa.assay.xlsx file below displays the best practice for annotating the mixing of experimental samples with a reference prior to LC/MS analysis.  
+:warning: Don't forget to rename your sheets according to the subprocess they are describing.
+
+## Connecting inputs and outputs
 
-![Spiking](./../img/ISA_AnnotationPattern_MixingSamples.svg)
+A key objective of the ARC is to trace each finding or result back to its specific biological experiment. Achieving this requires linking dataset files to their corresponding individual samples. To accomplish this, we follow a sequence of processes with defined inputs and outputs. Certain inputs and outputs may need to be reused or reproduced, while some processes may need to be applied to different inputs.
 
-By listing every raw data file twice, it becomes clear that the analyzed samples originated from the combination of an experimental sample and a reference, e.g. spiking of S1 with the reference resulted in the data file S1R.wiff.  
+This is a graph representing the structure of the ARC created for these examples. As you can see, samples flow seamlessly through studies and assays, ending with data files as an output. You should always aim for such connection of input and output throughout all isa tables. 
 
+<img src="./../img/mermaid-graph.drawio.svg" style="width:100%;display: block;margin: auto; padding: 30px 0px;">
```

Comment:
Can you please instead add the mermaid graph as a code block directly to the markdown? 
The knowledge base should be able to display that.

Reply:
i tried again but it's not working

Reply:
Sorry, did I tell you that you have to add it with the mermaid flag: 

```
```mermaid
graph LR

test1 --> test2

```

Reply:
> Sorry, did I tell you that you have to add it with the mermaid flag:
> 
> ```
> ```mermaid
> graph LR
> 
> test1 --> test2
> ```

I did but then in the html it's displayed as code block and not as a graph